plugins/inputs/snmp/snmp.go

```diff
@@ -535,8 +538,8 @@ func (t Table) Build(gs snmpConnection, walk bool) (*RTable, error) {
 		Time: time.Now(), //TODO record time at start
 		Rows: make([]RTableRow, 0, len(rows)),
 	}
-	for _, r := range rows {
-		if len(r.Tags) < tagCount {
+	for i, r := range rows {
+		if len(r.Tags)+emptyTags[i] < tagCount {
 			// don't add rows which are missing tags, as without tags you can't filter
 			continue
 		}
```

Comment:
There's no need to create a new PR. You can force-push the changes onto this branch and that will update this PR.